docs/pages/templates/chat/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 'Chat'
+---
+
+import { Cards } from 'nextra-theme-docs'
+import { Card } from '../../../components/Card'
+
+# Chat
+
+The chat template is a fully fledged chat application showcase the uses of the wallet-to-wallet messaging feature of the Liteflow infrastructure.
+
+<Cards num={2}>
+  <Card noPadding>![Home](/templates/chat/home.png)</Card>
+  <Card noPadding>![Home](/templates/chat/conversation.png)</Card>
+  <Card noPadding>![Home](/templates/chat/embed.png)</Card>
+  <Card noPadding>![Home](/templates/chat/widget.png)</Card>
+</Cards>
+
+You can have a look at a full demo of this template here: https://chat-liteflow.vercel.app/.
+
+## Features included
+
+<Cards>
+  <Card title="Wallet-to-wallet messaging" />
+  <Card title="Theme customization" />
+  <Card title="Embeddable widget" />
+</Cards>
+
```

Comment:
### Multi-Chain Compatibility
Our product seamlessly integrates with a variety of blockchain ecosystems to provide a versatile and expansive user experience. It is fully compatible with major chains including Ethereum, Polygon, Binance Smart Chain, and LightLink, as well as other EVM-compatible chains.